ruby_event_store/lib/ruby_event_store/mappers/protobuf.rb

```diff
@@ -81,7 +101,11 @@ def initialize(events_class_remapping: {})
         require_optional_dependency
         @pipeline = Pipeline.new([
           DomainEventProtoMapper.new,
+          ProtoMapper.new,
           EventClassRemapper.new(events_class_remapping),
+          SymbolizeMetadataKeys.new,
+          StringifyMetadataKeys.new,
```

Comment:
why both symbolize and stringify?

Reply:
to keep symmetric behaviour

Reply:
sounds like a weak argument for different pipelines ;)

Reply:
This is because edge mappers have different needs - `to_domain_event` mapper needs symbolized metadata keys, but `to_serialized_record` needs here stringified keys. Using both mappers here in that order solved problem.

Reply:
Exactly, with separate pipelines (one for dump and one for load) Symbolize would be in one but not the other. Same with Stringify.

Reply:
Yeah... but that's outdated code ;) This is no more in final version.